database-commons/src/main/java/io/cdap/plugin/db/DBErrorDetailsProvider.java

```diff
@@ -77,9 +77,10 @@ private ProgramFailureException getProgramFailureException(SQLException e, Error
       errorMessageWithDetails = String.format("%s For more details, see %s", errorMessageWithDetails,
```

Comment:
Please rephrase the `errorMessageWithDetails` above;

```
String.format("Error occurred in the phase: '%s' with sqlState: '%s', errorCode: '%s', errorMessage: %s", errorContext.getPhase(), sqlState, errorCode, errorMessage);
```

Reply:
This is just adding the doc link, the message is formatted above.

Reply:
yeah I know, I referred to the above errorMessage although unrelated to this PR but it would be good if we can fix it.

Reply:
Updated !